src/box/iproto_features.c

```diff
@@ -0,0 +1,64 @@
+/*
+ * SPDX-License-Identifier: BSD-2-Clause
+ *
+ * Copyright 2010-2021, Tarantool AUTHORS, please see AUTHORS file.
+ */
+#include "iproto_features.h"
+
+#include <msgpuck.h>
+#include <stdint.h>
+
+struct iproto_features IPROTO_CURRENT_FEATURES;
+
+uint32_t
+mp_sizeof_iproto_features(struct iproto_features *features)
```

Comment:
The argument could be `const`. The same in `mp_encode_iproto_features`.

Reply:
Fixed.